### ISSUE TITLE
Launcher name/icon

### ISSUE THREAD

**Issue by smichel17** [mpcjanssen/simpletask-android#851]
Having "ST Cloudless" as the launcher is weird to me, because I only use cloudless, so I don't really care what variant it is most of the time. I'd prefer if the name just read "SimpleTask".

That said, there should be something to distinguish them. I propose a variant on the icon, with a smaller icon in the lower right on top of the main ST icon. There are some concerns about using someone else's trademark, but I think I can find generic icons that work well enough. 

@mpcjanssen If you're in favor of this change, I'll go ahead and make the icons / send a PR.

**Comment by mpcjanssen** [comment]
Agreed on the app name trumping the flavour. Curious what the icons will look like.

**Comment by smichel17** [comment]
Something like this

![ic_launcher](https://user-images.githubusercontent.com/6563664/34643128-ddc2d9d8-f316-11e7-8f96-af608b1fc178.png)

**Comment by smichel17** [comment]
We can't use dropbox or nextcloud icons directly because they are both trademarked, but I don't think we need them.

I think we can/should set things up so there's only 2 build flavors: cloud and cloudless. The cloud flavor would allow you to choose between dropbox and nextcloud at runtime. We can separate the cloud flavor into two versions -- fdroid and play -- using build dimensions. The fdroid dimension won't include dropbox since that requires your api key to build it (unless there's a way to get dropbox working on f-droid, in which case we don't need to split it at all).

**Comment by smichel17** [comment]
For now, though, we can just use the cloud icon for both nextcloud and dropbox. Nextcloud isn't published anywhere, so if anybody is using it, they know what they installed. Or I can make a slightly different cloud, that's evocative of the nextcloud icon but clearly not the same (the real concern of the trademark is people mistaking this for an official product, so as long as it's sufficiently different, there's no big deal).

**Comment by mpcjanssen** [comment]
Nextcloud is in the playstore

**Comment by smichel17** [comment]
O.ok I guess I will need to think up another icon, then :)

**Comment by mpcjanssen** [comment]
Maybe a letter in the cloud? No cloud for cloudless N for nextcloud and D for dropbox. This also easily scales up to other cloude providers.

**Comment by smichel17** [comment]
I was thinking of that, but then:

- This doesn't scale well to non-English languages.
- I think it makes more sense to provide only one ST cloud app where you can change providers in the app, or maybe two (one with all providers and one without those that need an api key, for f-droid).

Still, I suppose the letter would work for now, and we could provide alternative resources for languages that dont use latin characters.

I'll think a little more and see if I can come up with anything better, then do the letter if not.

**Comment by mpcjanssen** [comment]
Will revert to Simpletask as name + letter.
